### PR TITLE
feat(testing): add rtl tests for components

### PR DESCRIPTION
**Pull request by LalaRukhQantas**
- Truncate
- ValueItem
- Wrapper

💁‍

## Related issues

💪

## Looks like this

🤨

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [ ] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), [Marcel](https://github.com/Anthropic), [Sam](https://github.com/SamuSan), [Long](https://github.com/KieraDOG)) or a Roo UI working-group human.
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [ ] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

File: src/components/ValueItem/ValueItem.test.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+test('renders the horizontal layout', async () => {
+  render(
+    <ValueItem {...defaultProps} layout="row">
+      There are limited time special offers available for this hotel for
+      specific dates or nights stay.
+    </ValueItem>,
+  );
+  expect(
+    screen.getByText('Extra special inclusions & great savings'),
```
Comment (paul-corbett-qantas) [inline src/components/ValueItem/ValueItem.test.jsx:22]: Maybe write as `screen.getByText(defaultProps.heading)` so only needs to be written once

**Review by paul-corbett-qantas — APPROVED** [review]

File: src/components/ValueItem/ValueItem.test.jsx (hunk 1 above)
Comment (LalaRukhQantas) [reply src/components/ValueItem/ValueItem.test.jsx]: Thanks @paul-corbett-qantas - It's done!

Comment (paul-corbett-qantas) [reply src/components/ValueItem/ValueItem.test.jsx]: 👍 Looks good!

**Pull request merged**